wallets/core/src/wallet.ts

```diff
@@ -308,8 +300,9 @@ class Wallet<InstanceType = any> {
     }
   }
 
-  setMeta(value: BlockchainMeta[]) {
-    this.meta = value;
+  setMeta(value: BlockchainMeta[], isContractWallet: boolean) {
```

Comment:
According to https://github.com/rango-exchange/rango-client/pull/427/files#r1382562861 , I suggest change this a little bit:
`setInfo(info: Info)`

Reply:
That would be great to let the user to pass partial values. something like `updateState(states: Partial<State>)` that we already have. It lets the user to only passing the needed values (`supportedBlockchains` or `isContractWallet`, and not both of them at the sametime).